app/assets/stylesheets/search.scss

```diff
@@ -76,12 +76,18 @@
   }
 }
 
-.search-project-name {
+.search-project-name h3 {
```

Comment:
Try this:
```
.search-project-name {
  height: 40px;
  text-align: left;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
}

.search-project-name h3 {
  display: inline;
}
```

Why?
- We make overflow to parent of the text, in the case `h3` is block element, so we make it inline to ensure that ellipse works fine for that
- With the initial code, not able to understand why you made it `70%` but with the initial code, even the name is `Public` it is getting ellipsed which ideally should not happen

<img width="1174" alt="Screenshot 2023-10-09 at 9 38 49 PM" src="https://github.com/CircuitVerse/CircuitVerse/assets/76901313/0e7138e6-86a2-4059-9c21-5fff17e19898">